core/editor/src/input/input_mapper.rs

```diff
@@ -6,6 +6,11 @@ use super::{
 	InputPreprocessor,
 };
 
+const NUDGE_AMOUNT: f64 = 1.;
+const NUDGE_AMOUNT_DIAG: f64 = NUDGE_AMOUNT * std::f64::consts::FRAC_1_SQRT_2;
```

Comment:
Sorry to contradict the earlier discussion from the team on Discord but a slower diagonal nudge distance is not actually a desirable feature because the user explicitly expects integer movements as multiples of 1 or 10 (while holding shift), so going 1.414 times faster diagonally is not really a problem compared to resulting in unpredictable, non-integer values for the final transform position. Even supporting simultaneous diagonal movement is an improvement over Photoshop's behavior, which just goes side-to-side or up-or-down depending on whether you hit the left/right/up/down key most recently while holding a diagonal pair.